server/text_generation_server/utils/flash_attn.py

```diff
@@ -178,7 +192,31 @@ def attention(
             False,
             None,
         )
-
+elif HAS_FLASH_ATTN_V2_ROCM and ROCM_USE_FLASH_ATTN_V2_TRITON:
+    
+    def attention(
+        q,
+        k,
+        v,
+        out,
+        cu_seqlens,
+        max_s,
+        softmax_scale,
+        window_size_left=-1,
+    ):
+        output, _ = triton_attention(
+            q,
+            k,
+            v,
+            out,
+            cu_seqlens,
+            cu_seqlens,
+            max_s,
+            max_s,
+            True,
```

Comment:
FYI, Gemma/PaliGemma now use a parameter `causal` (because paligemma uses non causal attention during prefill).

Reply:
We haven't updated every path yet, just making `causal=True` as a kwarg should suffice.

Reply:
good catch